OWNERS

```diff
@@ -7,4 +7,8 @@ reviewers:
   - leodido
   - fntlnz
   - maxgio92
+<<<<<<< HEAD
   - jonahjon
+=======
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 12: could not find expected ':'.